modules/iam-role-for-service-accounts-eks/policies.tf

```diff
@@ -331,6 +331,17 @@ data "aws_iam_policy_document" "ebs_csi" {
       resources = var.ebs_csi_kms_cmk_ids
     }
   }
+
+  # https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/docs/fast-snapshot-restores.md#prerequisites
+  dynamic "statement" {
+    for_each = var.ebs_csi_fast_snapshot_restore ? [1] : []
+    content {
+      actions = [
+        "ec2:EnableFastSnapshotRestores"
```

Comment:
We can just add this permission into https://github.com/terraform-aws-modules/terraform-aws-iam/blob/f65532fc786b79b1ef24b0993e8f2ff6b25b22cf/modules/iam-role-for-service-accounts-eks/policies.tf#L157-L168

we can skip the variables and dynamic statement, etc.

Reply:
Done